include/odp/api/spec/packet_io.h

```diff
@@ -845,6 +853,37 @@ typedef struct odp_pktio_capability_t {
 		uint32_t max_output;
 	} maxlen;
 
+	/** Supported packet Tx completion options */
+	struct {
+		/**
+		 * Scheduled queue support
+		 *
+		 * This defines whether schedule queues are supported for receiving Tx
+		 * completion events.
+		 *
+		 * 0: Scheduled queues are not supported for receiving Tx completion events.
+		 * 1: Scheduled queues are supported for receiving Tx completion events.
+		 * @see odp_packet_tx_compl_request()
+		 */
+		odp_bool_t queue_type_sched;
+
+		/**
+		 * Plain queue support
+		 *
+		 * This defines whether plain queues are supported for receiving Tx
+		 * completion events.
+		 *
+		 * 0: Plain queues are not supported for receiving Tx completion events.
+		 * 1: Plain queues are supported for receiving Tx completion events.
+		 * @see odp_packet_tx_compl_request()
+		 */
+		odp_bool_t queue_type_plain;
+
+		/** ODP_PACKET_TX_COMPL_ALL supported */
+		uint32_t all_xmit_or_drop:1;
```

Comment:
Maybe mode_all (or mode.all) is better name for this flag, so that the flag names would match mode enum values.

Reply:
Changed it to mode_all in V6.